client/my-sites/email/email-management/email-home.tsx

```diff
@@ -29,15 +30,20 @@ import type { ReactNode } from 'react';
 
 import './style.scss';
 
-const ContentWithHeader = ( props: { children: ReactNode } ) => {
+interface ContentWithHeaderProps {
+	children: ReactNode;
+	className?: string;
+}
```

Comment:
```suggestion
type ContentWithHeaderProps = React.PropsWithChildren< {
	className?: string;
} >;
```
Optional nit. Helps keep the `children` type definition correct.

Reply:
Corrected ✅